impress-org/givewp-next-gen
# Feature: add "default level" option to the Donation amount and levels block

## Description



This PR adds a new "Default Level" property as a "Radio" setting on the side of each level input. Now is also possible to reorder the levels using the mouse to drag the items.

## Affects



The Donation amount and levels block

## Visuals



![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/54572c5d-af34-4776-9664-6fbb6f574530)

![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/5ffc73f0-f472-40fc-b4f8-edb4fbd86c4f)


## Testing Instructions



1. On the DONATION OPTION, set the "Multi-level Donation" value;
2. Go to the "Donation Levels" section and you could see a "Radio" setting on the side of each level;
3. Test the new controls by choosing different default levels and removing or adding new ones. 

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204282271232272
